docs/DeveloperGuide.md

```diff
@@ -432,7 +432,7 @@ This feature creates a `Remark` instance which is stored internally in Nuudle as
 which is in turn stored in the `PatientBook`. These classes are a part of the `Model` component and are illustrated
 in the class diagram below. 
 
-![RemarkLogicClassDiagram](images/RemarkLogicClassDiagram2.png)
+![RemarkLogicClassDiagram](images/RemarkLogicClassDiagram.png)
```

Comment:
Thanks for fixing this! Was gonna change it later cause I was consulting our TA abt it too. Apparently, Prof accepts Draw.io diagrams too haha but nonetheless using plantUML is a much safer choice